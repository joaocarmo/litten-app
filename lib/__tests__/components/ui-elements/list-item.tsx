import TestRenderer from 'react-test-renderer'
import ThemeProvider from '@components/theme/provider'
import { UIListItem } from '@ui-elements'

describe('Snapshot test for the "UIText" component', () => {
  it('renders correctly', () => {
    const element = TestRenderer.create(
      <ThemeProvider>
        <UIListItem badgeNum={2} hasExtra>
          Some text here
        </UIListItem>
      </ThemeProvider>,
    ).toJSON()
    expect(element).toMatchSnapshot()
  })
})
