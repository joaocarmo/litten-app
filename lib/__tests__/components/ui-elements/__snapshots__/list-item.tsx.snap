// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for the "UIText" component renders correctly 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "#ffffff",
        "borderRadius": 6,
        "flex": 1,
        "marginBottom": 8,
        "marginTop": 8,
        "minHeight": 64,
        "overflow": "hidden",
        "paddingLeft": 20,
        "paddingRight": 20,
      },
      {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "space-between",
      },
      undefined,
      undefined,
      undefined,
    ]
  }
>
  <View
    style={
      {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "flex-start",
      }
    }
  >
    <Text
      style={
        [
          {
            "color": "#000000",
            "fontSize": 14,
            "fontWeight": "500",
          },
          undefined,
          undefined,
          undefined,
        ]
      }
    >
      Some text here
    </Text>
  </View>
  <View
    style={
      {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        {
          "alignItems": "center",
          "backgroundColor": "#eff1f3",
          "borderRadius": 8,
          "height": 20,
          "justifyContent": "center",
          "marginRight": 4,
          "minWidth": 20,
          "padding": 4,
        }
      }
    >
      <Text
        style={
          [
            {
              "color": "#000000",
              "fontSize": 9.832647462277093,
              "fontWeight": "700",
            },
            undefined,
          ]
        }
      >
        2
      </Text>
    </View>
    <SvgMock
      fill="#474545"
      height={12}
      width={12}
    />
  </View>
</View>
`;
